java/src/jmri/jmrit/throttle/ThrottleBundle_fr.properties

```diff
@@ -52,7 +52,7 @@ ControlPanelProperties = Propri\u00e9t\u00e9s
 TitleEditSpeedControlPanel = Modifier Panneau de commande de vitesse
 CheckBoxTrackSliderInRealTime = Curseur de vitesse en temps r\u00e9el
 CheckBoxHideSpeedStepSelector = Masquer le s\u00e9lecteur du nombre de crans de vitesses
-ButtonDisplaySpeedSliderContinuous = Affichage du curseur de vitesse man\u0153uvres (-100 \u00e0 0 de 100)
+ButtonDisplaySpeedSliderContinuous = Affichage du curseur de vitesse man\u0153uvres (-100 \u00e0 0 \u00e0 100)
```

Comment:
You can directly put UTF characters in properties files now if you want to.  No need to recode them with \u